rmf_traffic_ros2/src/rmf_traffic_ros2/convert_Graph.cpp

```diff
@@ -325,7 +340,7 @@ rmf_traffic::agv::Graph json_to_graph(
     }
 
     // dock_name is only applied to the lane going to the waypoint, not exiting
-    const rmf_traffic::Duration duration = std::chrono::seconds(5);
+    const rmf_traffic::Duration duration = std::chrono::nanoseconds(5);
```

Comment:
I'm sure the choice of 5 second docking times was an arbitrary default, but I think 5ns is even more arbitrary. At that point we may as well make the default 0ns.

@luca-della-vedova do you have anything that depends on a 5 second default duration for docking? If not, I suggest we change the default to 0s.

Reply:
No problem for my side! The 5 seconds was taken from [parse_graph](https://github.com/open-rmf/rmf_ros2/blob/main/rmf_fleet_adapter/src/rmf_fleet_adapter/agv/parse_graph.cpp#L286) so I guess to keep the implementation consistent all of them should have the same default?

Reply:
Whoops that diff is an unintended consequence of search and replace. I can revert it back to 5 seconds?